src/coreclr/jit/fgopt.cpp

```diff
@@ -769,7 +769,6 @@ bool Compiler::fgRemoveDeadBlocks()
 // Notes:
 //   Each block's `bbPreorderNum` and `bbPostorderNum` is set.
 //   The `fgBBReversePostorder` array is filled in with the `BasicBlock*` in reverse post-order.
-//   This algorithm only pays attention to the actual blocks. It ignores any imaginary entry block.
```

Comment:
I wasn't totally sure what this was referring to... but today we do ensure all blocks are visited below, so it seemed odd.

Reply:
SSA creates a stack-allocated BB0 when it computes dominators, I suspect this is the "imaginary entry block."

Reply:
Hmm, might me. Still, not sure it is too relevant for this computation.